relay-server/src/actors/spooler/mod.rs

```diff
@@ -1074,11 +1074,9 @@ mod tests {
             sender: tx.clone(),
         });
 
-        tokio::time::sleep(Duration::from_millis(500)).await;
-
         // There are enough permits, so get an envelope:
-        let res = rx.try_recv();
-        assert!(res.is_ok(), "{res:?}");
+        let res = rx.recv().await;
```

Comment:
Is there a timeout for a test to run? Is there a way to know if a test takes way too long, besides looking at the CI time?

I'm curious, no need to take any action.